### DISCUSSION TITLE
How to use command line with 128GiB RAM - Linux - Windows

### DISCUSSION THREAD

**Discussion by casa10** [madMAx43v3r/chia-plotter#240] [General]
if you 128Gib of ddr ram :
- using Linux version, how to specify to use ddr ram and not NVME/SSD/HDD as temporary folder (tmp2 of 110 GiB) ? For Linux, which type choose (tmpfs or ramfs)
- using windows, same question ?
 Thank you

**Comment by polkadotbabe** [comment]
Super easy in any unix, tmpfs is a filesystem that lives in RAM, so as root  (either sudo or su -):

mkdir /mnt/ramdisk
mount -t tmpfs -o size=111G tmpfs /mnt/ramdisk

That'd give you the required size for tmp2 ramdisk. Super fast, no wear, leaving about 16G RAM left, which is enough to run chia_plot. Which you'd run like this from a script -- screen gives you another terminal window -- if just pasting it to the command line replace $(nproc) with echo "$(nproc)" or your total cpu thread count:

#!/bin/bash
screen -dmS chia1 /home/chia/chia_plot -r $(nproc) -n -1 -d /home/chia/ChiaP/ -p YOURPOOLKEY \
-f YOURFARMKEY -t /home/chia/temp1/ -2 /mnt/ramdisk/ 4 |tee /home/chia/chia1.log &

Get into the screen with "screen -Dr chia1" and out of it with ctrl-a-d

In windows: Wipe and install linux, follow along ^^^^   ;-)

**Reply by casa10** [reply]
Thank you already for all :)
